src/engine/init.c

```diff
@@ -486,6 +486,51 @@ abt_init(int argc, char *argv[])
 		return dss_abterr2der(rc);
 	}
 
+#ifdef ULT_MMAP_STACK
```

Comment:
Use the 'switch' instead of 'ifdef'?

Reply:
Never mind, I mean the 'ifdefs' could be replaced by the env variable switch, but you've answered this in previous comment.